### ISSUE TITLE
Open source licensing

### ISSUE THREAD

**Issue by pudu1991** [jgthms/web-design-in-4-minutes#7]
Is this project open source? if so, what's license for it?

**Comment by phikal** [comment]
I'd recommend something like creative commons, since the most important part of the project is the content, not the code.

**Comment by chocochino** [comment]
I agree with @phikal. CC-BY 4.0 would suit this project, since the project was meant to be a tutorial.
